ui/userguide/UserGuideBody.jsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+
+import ShortcutTable from '../shared/ShortcutTable';
+
+import logviewerIconHelp from '../img/logviewerIconHelp.svg';
+
+const possiblesNotation = [
+  {
+    text: '+n',
+    classes: 'btn-green ug-btn-comment',
+    explanation: 'Collapsed job count',
+  },
+  {
+    text: 'Th()',
+    classes: 'btn-dkgray ug-btn-comment',
+    explanation: 'Wrapped job group',
+  },
+  {
+    classes: 'btn-orange-classified autoclassified ug-btn-comment',
+    explanation: 'Hollow asterisk, auto-classified',
+  },
+  {
+    classes: 'btn-orange-classified ug-btn-comment',
+    explanation: 'Asterisk, classified',
+  },
+  {
+    classes: 'btn-ltgray ug-btn-bg',
+    explanation: 'Light gray, pending',
+  },
+  {
+    classes: 'btn-dkgray ug-btn-bg',
+    explanation: 'Gray, running',
+  },
+  {
+    classes: 'btn-green ug-btn-bg',
+    explanation: 'Green, success',
+  },
+  {
+    classes: 'btn-orange ug-btn-orange',
+    explanation: 'Orange, tests failed',
+  },
+  {
+    classes: 'btn-purple ug-btn-purple',
+    explanation: 'Purple, infrastructure exception',
+  },
+  {
+    classes: 'btn-red ug-btn-red',
+    explanation: 'Red, build error',
+  },
+  {
+    classes: 'btn-dkblue ug-btn-bg',
+    explanation: 'Dark blue, build restarted',
+  },
+  {
+    classes: 'btn-pink ug-btn-bg',
+    explanation: 'Pink, build cancelled',
+  },
+  {
+    classes: 'btn-yellow ug-btn-yellow',
+    explanation: 'Yellow, unknown',
+  },
+  {
+    classes: 'btn-ltblue ug-btn ug-btn-bg',
+    explanation: 'Light blue, superseded',
+  },
+];
+
+class UserGuideBody extends React.Component {
+  componentWillMount() {
```

Comment:
`componentWillMount()` is deprecated and no longer recommended:
https://reactjs.org/docs/react-component.html#unsafe_componentwillmount

That said, I don't think we really need to have dynamic dates showing, so can drop that feature? ie: Adjust their usage in `render()` to be a fixed `YYYY-MM-DD` string?

This component could then also be made a functional component rather than a class.

Reply:
Yes, I read recently that the WillMount should changed by DidMount, but how you specified, this component can be a functional component as it's not using life cycle